src/components/IFrame.js

```diff
@@ -92,6 +111,11 @@ export default function ReportScreen() {
         });
     }, []);
 
+    useEffect(() => {
+        document.cookie = `authToken=${session.authToken}; domain=expensify.com.dev; path=/;`;
+        document.cookie = `email=${session.email}; domain=expensify.com.dev; path=/;`;
```

Comment:
Not sure if it's worth but maybe we can early return when either of authToken or email doesn't exist?

Reply:
I don't think there is a situation when these values are not defined because IFrame component is displayed only if the user is logged in 🤔 

or do you think we're missing any exceptions to this?

Reply:
ok it's fine if no case